<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notepad</class>
 <widget class="QWidget" name="notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>561</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Enrollment">
    <attribute name="title">
     <string>Enrollment</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>User Info</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Userimage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="Userchooser">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>371</width>
        <height>29</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>(Choose from list)</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="UpdateInfo">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearInfo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UploadImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>129</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload Photo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Age:   </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Userage">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QComboBox" name="Usersex">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sex:   </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>146</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string extracomment="Enter your name"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>26</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pass:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Username">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>147</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string extracomment="Enter your name"/>
      </property>
     </widget>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>Userchooser</zorder>
     <zorder>UpdateInfo</zorder>
     <zorder>ClearInfo</zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder>UploadImage</zorder>
     <zorder>label_3</zorder>
     <zorder>Userage</zorder>
     <zorder>label_4</zorder>
     <zorder>Password</zorder>
     <zorder>label_5</zorder>
     <zorder>label_2</zorder>
     <zorder>Username</zorder>
     <zorder>Usersex</zorder>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>451</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Voice Enrollment</string>
     </property>
     <widget class="QLabel" name="Animation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="enroll">
         <property name="text">
          <string>Enroll!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startTrain">
         <property name="text">
          <string>Start Traning!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="enrollRecord">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopEnrollRecord">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="enrollTime">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Recognition">
    <attribute name="title">
     <string>Test Recognition</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>411</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>Recognition</string>
     </property>
     <widget class="QLabel" name="Animation_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>131</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="recoUserImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="recoUsername">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="recoTime">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="Alading">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="recoRecord">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopRecoRecord">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>Animation_2</zorder>
     <zorder>verticalLayoutWidget_3</zorder>
     <zorder>recoTime</zorder>
     <zorder>Alading</zorder>
     <zorder>horizontalLayoutWidget_6</zorder>
     <zorder>recoUserImage</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
